src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
the type is as PG mentioned is used for a very simple auto validation but sadly it is not used for the input field. Which means we have to provide a string which is then validated if it is a number. There is definitively some refactoring possible.